### ISSUE TITLE
alluxio.user.file.metadata.sync.interval not working when mounting a whole bucket as primary UFS

### ISSUE THREAD

**Issue by cangyin** [Alluxio/alluxio#15298]
**Alluxio Version:**
2.7.3

**Describe the bug**
The `alluxio.user.file.metadata.sync.interval` is not working when mounting a whole bucket as _primary UFS_, but works well if mounting a directory in the bucket, or if mounting a bucket (or directory in bucket) as _additional UFS_.
As documented [here](https://docs.alluxio.io/os/user/2.7.3/en/ufs/S3.html#:~:text=Note%20that%20if%20you%20want%20to%20mount%20the%20whole%20s3%20bucket).

**To Reproduce**

Good case:
1. start a alluxio cluster with configuration as below
  ```ini
  # Common properties
  alluxio.master.hostname=<master-host-ip>
  alluxio.master.mount.table.root.ufs=s3://alluxio-ufs/data/
  aws.accessKeyId=<key>
  aws.secretKey=<secret>
  alluxio.underfs.s3.endpoint=<s3-endpoint-url>
  alluxio.underfs.s3.disable.dns.buckets=true
  alluxio.underfs.s3.signer.algorithm=S3SignerType
  # User properties
  alluxio.user.file.metadata.sync.interval=1m
  ```
2. upload a new file into  s3://alluxio-ufs/data/file1
3. `./bin/alluxio fs -Dalluxio.user.file.metadata.sync.interval=0 ls -R /`,  the `file1` appeared as expected.

Bad case:
1. start a alluxio cluster with configuration as below
  ```ini
  ...
  alluxio.master.mount.table.root.ufs=s3://alluxio-ufs/ # mount the whole bucket.
  ...
  ```
2. upload a new file into  s3://alluxio-ufs/file1
3. `./bin/alluxio fs -Dalluxio.user.file.metadata.sync.interval=0 ls -R /`,  the `file1` did not appear.

**Expected behavior**
as detailed above.

**Urgency**
with little urgency.

**Are you planning to fix it**
no

**Comment by Jackson-Wang-7** [comment]
Is this still the case? I have reproduced this case, and it's working with `alluxio.user.file.metadata.sync.interval`.

**Comment by dechoma** [comment]
I got a similar issue when using alluxio versions `2.7.2 - 2.7.4`  no metadata when mounted whole s3 bucket and on s3 side some `Object name invalid` errors were present. I was unable to reproduce this on versions  `2.7.1` or` 2.8.0+`

below is my mount test
```
#!/bin/bash
echo "---alluxio Version---"
alluxio version
alluxio fs mount --readonly --option alluxio.underfs.s3.endpoint=minio:9000 --option aws.accessKeyId=admin --option aws.secretKey=adminadmin --option alluxio.underfs.s3.disable.dns.buckets=true /s3mount s3a://alluxio
echo "alluxio fs ls /s3mount -> listing content of /s3mount"
alluxio fs ls /s3mount
alluxio fs unmount /s3mount
echo "------done--------"
```
and the output:
![Screenshot 2022-08-16 at 09 28 27](https://user-images.githubusercontent.com/29060284/184851703-affc22a1-bf26-4da4-9e19-d529dbcd17d4.png)
![Screenshot 2022-08-16 at 09 30 42](https://user-images.githubusercontent.com/29060284/184851715-45ee9d16-e715-4464-a4e0-0b38d15f89b3.png)
`

**Comment by dechoma** [comment]
issue was fixed in`2.8.0` by this pr https://github.com/Alluxio/alluxio/pull/15361

**Comment by cangyin** [comment]
@dechoma awesome!
